File: lat3_PHP/Ram.php
<?php

class Ram{
    private $Capacity = "";
    private $Price = 0;
    
    public function __construct($Capacity = "", $Price = 0){
        $this->Capacity = $Capacity;
        $this->Price = $Price;
    }

    public function setCapacity($Capacity){
        $this->Capacity = $Capacity;
    }

    public function getCapacity(){
        return $this->Capacity;
    }

    public function setPrice($Price){
        $this->Price = $Price;
    }

    public function getPrice(){
        return $this->Price;
    }

    public function printRAM(){
        echo "RAM Capacity      : ". $this->getCapacity() ."<br>";
        echo "RAM Price         : ". $this->getPrice() ."<br>";
    }

}  
?>

File: lat3_java/ram.java
public class ram {
    private String capacity;
    private int price;

    public ram() {
    }

    public ram(String capacity, int price) {
        this.capacity = capacity;
        this.price = price;
    }

    public void setCapacity(String capacity) {
        this.capacity = capacity;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public int getPrice() {
        return price;
    }

    public String getCapacity() {
        return capacity;
    }
    
    public void printRam(){
        System.out.print("RAM             : ");
        System.out.println(getCapacity());
        System.out.print("RAM Price       : ");
        System.out.println(getPrice());
    }
    
}